packages/core/stories/card/card.stories.tsx

```diff
@@ -166,30 +170,33 @@ export const DefaultWithLinkAndImage: StoryFn<typeof Card> = () => (
       className="card-demo-image"
       style={{ width: "-webkit-fill-available" }}
     />
-    <div
+    <StackLayout
       style={{
         padding:
           "0px var(--salt-size-container-spacing) var(--salt-size-container-spacing) var(--salt-size-container-spacing)",
```

Comment:
--salt-size-container-spacing was deprecated and replaced with --salt-spacing-300